dev-docs/bidder-adaptor.md

```diff
@@ -12,259 +11,504 @@ nav_section: adapters
+
+1. The `getUserSyncs` callback of the `BaseAdapter` model
+2. The `userSync.registerSync` function
 
 {% highlight js %}
-var bidRequest  = utils.getBidRequest(id);
-var bidResponse = bidfactory.createBid(STATUS.NO_BID, bidRequest);
+
+{
+    getUserSyncs: function(syncOptions, serverResponses) {
+        const syncs = []
+        if (syncOptions.iframeEnabled) {
+            syncs.push({
+                type: 'iframe',
+                url: '//acdn.adnxs.com/ib/static/usersync/v3/async_usersync.html'
+            });
+        }
+        if (syncOptions.pixelEnabled && serverResponses.length > 0) {
+            syncs.push({
+                type: 'image',
+                url: serverResponses[0].body.userSync.url
+            });
+        }
+        return syncs;
+    }
+}
+
 {% endhighlight %}
 
-Example:
+## Supporting Video
+
+Follow the steps in this section to ensure that your adapter properly supports video.
+
+### Step 1: Register the adapter as supporting video
+
+Add the `supportedMediaTypes` argument to the spec object, and make sure `video` is in the list:
 
 {% highlight js %}
 
-// In the bidder's API callback...
+export const spec = {
+    code: BIDDER_CODE,
+    supportedMediaTypes: ['video'],
+    ...
+}
+
+{% endhighlight %}
+
+### Step 2: Accept video parameters and pass them to your server
 
-// the bidder API's ad response unit that has info like CPM, creative content
-var adUnit;
+See the [AppNexus AST adapter]({{site.baseurl}}/dev-docs/bidders.html#appnexusAst) for an example of how
+video parameters may be passed in from the ad unit.
 
-var bidRequest  = utils.getBidRequest(id);
-var bidObject = bidfactory.createBid(STATUS.GOOD, bidRequest);
-bidObject.bidderCode = 'openx';
-bidObject.cpm = Number(adUnit.get('pub_rev')) / 1000;
-bidObject.ad = adUnit.get('html');
-bidObject.width = adUnit.get('width');
-bidObject.height = adUnit.get('height');
+### Step 3: Respond with VAST or a VAST URL
 
-// send the bidResponse object to bid manager with the adUnitCode.
-bidmanager.addBidResponse(adUnitCode, bidObject);
+When the bidder returns VAST or a VAST URL in its bid response, it needs to add the result into either `bid.vastXml` or `bid.vastUrl`. For example, here is some [code from the Tremor adapter](https://github.com/prebid/Prebid.js/blob/master/modules/tremorBidAdapter.js#L142) showing how it's done:
 
-// invalid bid response
-bidObject = bidfactory.createBid(STATUS.NO_BID, bidRequest);
-bidObject.bidderCode = 'openx';
-bidmanager.addBidResponse(adUnitCode, bidObject);
+{% highlight js %}
+
+function createBid(status, reqBid, response) {
+    let bid = bidfactory.createBid(status, reqBid);
+
+    if (response) {
+        bid.cpm = response.price;
+        bid.crid = response.crid;
+        bid.vastXml = response.adm;
+        bid.mediaType = 'video';
+    }
+
+    return bid;
+}
 
 {% endhighlight %}
 
-**`adUnitCode` in `addBidResponse`**
+#### If you would like to participate in auctions for outstream video ads
```

Comment:
Support for the outstream video context should either be its own section under "Supporting Video" or we should have a separate step/ section of step 2 which describes properly ingesting the video context from the request.

Reply:
Took your suggestion to move this info into subsections of Step 2 in 5bf5a29 ('Ingesting the Video Context', and 'Outstream Video Renderers').  The content is basically the same as before, just reorganized into those sections.  Let me know if you have any additional feedback.